### PR TITLE
fixed a grammar error in readme.md

### PR DESCRIPTION
**Pull request by MeetRajput00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ For a good overview and starting point, please check out the official [Solidity
 
 Solidity is a statically-typed curly-braces programming language designed for developing smart contracts
 that run on the Ethereum Virtual Machine. Smart contracts are programs that are executed inside a peer-to-peer
-network where nobody has special authority over the execution, and thus they allow to implement tokens of value,
+network where nobody has special authority over the execution, and thus they allow you to implement tokens of value,
```
Comment (leonardoalt) [inline README.md]: Agree that it's missing here, but can you maybe change it to `anyone`?

**Review by leonardoalt — CHANGES_REQUESTED** [review]
Thanks for the PR! Please rebase your branch onto origin/develop instead of merging to get rid of the merge commit.

File: README.md (hunk 1 above)
Comment (MeetRajput00) [reply README.md]: Sure

**Comment by MeetRajput00** [comment]
> Thanks for the PR! Please rebase your branch onto origin/develop instead of merging to get rid of the merge commit.

Okay

**Pull request closed without merging**